### PR TITLE
Add new nuget publish for Utils.Testing to Action

### PR DESCRIPTION
**Pull request by carlos-takeapps**

### CODE REVIEW

**Review by monsieurleberre — APPROVED** [review]

File: .github/workflows/publish.nuget.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@ on:
 
 env:
   SOLUTION_PATH: "src/Trakx.Utils.sln"
-  PROJECT_PATH: "src/Trakx.Utils/Trakx.Utils.csproj"
+  UTILS_PROJECT_PATH: "src/Trakx.Utils/Trakx.Utils.csproj"
+  TEST_PROJECT_PATH: "src/Trakx.Utils/Trakx.Utils.Testing.csproj"
```
Comment (monsieurleberre) [inline .github/workflows/publish.nuget.yml:17]: In our others repos, this should be `src/Trakx.Utils.Testing/Trakx.Utils.Testing.csproj`

**Review by monsieurleberre — CHANGES_REQUESTED** [review]
thinking about it now, if we merge that PR in first, but have no code behind, it will simply fail...
can you please submit the rest of the code we need for that to work?

File: .github/workflows/publish.nuget.yml (hunk 1 above)
Comment (carlos-takeapps) [reply .github/workflows/publish.nuget.yml]: I fixed the path in a new commit

**Comment by carlos-takeapps** [comment]
> thinking about it now, if we merge that PR in first, but have no code behind, it will simply fail...
> can you please submit the rest of the code we need for that to work?

I created a PR from the branch that has the new project

**Comment by carlos-takeapps** [comment]
If we can merge this branch to dev, we will have the new publish package task available from the dev branch.

**Review by monsieurleberre — APPROVED** [review]

**Pull request merged**